components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -967,26 +967,14 @@
 .fuzzy-date-picker select {
   font-family: Arial,Verdana,sans-serif;
 }
-.fuzzy-date-picker .year {
-  font-size: 14px;
-  max-width: 68px;
+.fuzzy-date-picker input[type=text].invalid {
+  border: 1px solid #c33;
 }
-.fuzzy-date-picker .month {
-  font-size: 14px;
-  max-width: 48px;
-}
-.fuzzy-date-picker .day {
-  font-size: 14px;
-  max-width: 48px;
-}
-.fuzzy-date-picker .empty {
-  font-size: .7em;
-  opacity: .5;
+.fuzzy-date-picker input[type=text].text {
```

Comment:
you don't have any non-`[type=text]` inputs, should we still be getting super specific with the `.text` class selector?